opencti-platform/opencti-front/src/private/components/nav/TopBar.tsx

```diff
@@ -496,8 +497,17 @@ const TopBarComponent: FunctionComponent<TopBarProps> = ({
                 {t('Profile')}
               </MenuItem>
               <MenuItem onClick={handleOpenDrawer}>{t('Feedback')}</MenuItem>
-              <MenuItem id="logout-button" onClick={() => handleLogout()}>
```

Comment:
You can maybe delete the "handleLogout" function at the same time ?

Reply:
it's actually a global function exported and used elsewhere. 
it's a simple `window/location = "/logout"`

Reply:
Actually used only in `TimeoutLock.tsx`, maybe move it here?

Reply:
yeah let's do that